ximena_subieta/space_challenge/src/Item.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * This class represents an item to be carried by the rockets.
+ */
+public class Item {
+    String name;
```

Comment:
assign access (public, private or protected)
private String name;

Reply:
I thought if there is no access modifier set it has public access. But I don't mind updating it.